tcb.spec

```diff
@@ -87,8 +87,12 @@ rmdir /sbin/chkpwd.d
 %_includedir/tcb.h
 %_libdir/libtcb.a
 %_libdir/libtcb.so
+%_libdir/pkgconfig/tcb.pc
 
 %changelog
+* Thu Sep 30 2021 Björn Esser <besser82@fedoraproject.org> - 1.2-owl1.1
```

Comment:
I think we should go with version 1.3. Updates of RPM spec `Release` are for changes that are limited to the RPM packaging only, whereas we're changing plenty of things in the codebase since 1.2.

Reply:
Will fix that in a minute; I'll leave bumping the version in the spec file up to you on release.